### PR TITLE
 VPN pricing experiment on welcome 10 [fix #10257] 

### LINKED ISSUE #10257: Update moments page for VPN
## Description

PMM team has requested the following updates to https://www.mozilla.org/en-US/firefox/welcome/10/

1. Update headline. Copy to be provided by creative team
2. Update CTA to link to pricing section of landing page: https://www.mozilla.org/en-US/products/vpn/?source=welcome10#pricing

UPDATE:
This will run as an a/b test with traffic cop.

### Both Variations:

**Logo:** VPN lockup logo (not just icon)

**CTA text:** Get Mozilla VPN

### Variation 1 (50% traffic)

**Headline:** Ending soon: $4.99/month for a secure connection on every device

**Paragraph:** Protect your entire internet connection — on your computer, your tablet, and even your phone—with this limited-time offer.

### Variation 2 (50% traffic)

**Headline:** Blur your location online for next-level privacy

**Paragraph:** Get Mozilla VPN for $4.99/month — this offer ends soon. Start now.

---

#### Success Criteria
- [ ] Able to provide signup attribution for users who landed on this moments page
- [ ] Able to track and compare CTR of each variation.
- [ ] Able to only send users from EN countries that have access to VPN and can buy in USD

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Testing two copy variants on /firefox/welcome/10/ with the traffic cop behind the switch `welcome-10-experiment`

We should be able to target by region in-product so there's no geolocation check on this page. There's always a chance someone would go to the URL directly (like if someone shares it), but that's deemed to be an edge case. Such people would see the US$4.99 price but upon clicking through to the landing page would see the price for their region (or the wait list). Some countries are charged in USD anyway so the price is correct.

Users who don't get enrolled in the experiment (because they block the script or have DNT enabled) fall back to the old page. Those who do get enrolled are a 50/50 split between the two new variants. So the old page may still get some signups but they can be filtered out of any experiment analysis, or we can even treat that something like a control and see how many people click without the price or copy changes.

## Issue / Bugzilla link
#10257 

## Testing
http://localhost:8000/en-US/firefox/welcome/10/?v=1
http://localhost:8000/en-US/firefox/welcome/10/?v=2

### CODE REVIEW

**Comment by craigcook** [comment]
@alexgibson This was put together somewhat hastily and you know far more about the VPN stuff, so it's very likely I've overlooked something or made a dumb mistake.

File: bedrock/firefox/templates/firefox/welcome/page10.html [hunk 1]
```diff
@@ -12,52 +10,82 @@
...
-{% set _source = '?source=welcome10' %}
+
+{% if variation == '1' %}
+  {% set title = 'Get a <strong>secure connection</strong> on every device'|safe %}
+  {% set desc = 'Protect your internet connection — on your computer, your tablet and even your phone — with this limited-time offer.' %}
+  {% set source = '?source=welcome10-exp-v1' %}
+  {% set cta_text = 'Get Mozilla VPN' %}
+{% elif variation == '2' %}
+  {% set title = 'Blur your location online for <strong>next-level privacy</strong>'|safe %}
+  {% set desc = 'Get Mozilla VPN for ' + vpn_monthly_price(plan='12-month', lang=LANG) + ' — this limited-time offer ends soon.' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/welcome/page10.html:33]: You're not using any geo-location here, so I don't think `vpn_monthly_price(plan='12-month', lang=LANG)` is really needed and you could straight up hard code `US$4.99/month`?

This is also using the 12-month plan price, which isn't yet available in the US (even though the monthly monetary value is the same).

File: bedrock/firefox/templates/firefox/welcome/page10.html [hunk 2]
```diff
@@ -12,52 +10,82 @@
...
+  <section class="c-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+    {% if variation in ['1', '2'] %}
+      <img class="c-main-image" src="{{ static('protocol/img/logos/mozilla/vpn/logo-word-hor-stack.svg') }}" width="200" alt="">
+    {% else %}
+      <img class="c-main-image" src="{{ static('protocol/img/logos/mozilla/vpn/logo-flat.svg') }}" width="90" alt="">
+    {% endif %}
+
+    {% if variation == '1' %}
+      <p class="c-pre-title">{{ vpn_monthly_price(plan='12-month', lang=LANG) }} — offer ends soon</p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/welcome/page10.html:55]: Same as above, you could just hardcode the price here?

File: bedrock/firefox/templates/firefox/welcome/page10.html [hunk 3]
```diff
@@ -12,52 +10,82 @@
 
 {% block body_class %}welcome-page10{% endblock %}
 
+{% block experiments %}
+  {% if switch('welcome-10-experiment', ['en-US', 'en-CA']) %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/welcome/page10.html:14]: Should `en-GB` be added here? (I noticed in the experiment view there are three English locales listed).

File: media/js/firefox/welcome/welcome10-experiment.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+    /* update dataLayer with experiment info */
+    var href = window.location.href;
+
+    var initTrafficCop = function () {
+        if (href.indexOf('v=') !== -1) {
+            if (href.indexOf('v=a') !== -1) {
```
Comment (alexgibson) [inline media/js/firefox/welcome/welcome10-experiment.js:12]:
```suggestion
            if (href.indexOf('v=1') !== -1) {
```

File: media/js/firefox/welcome/welcome10-experiment.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                window.dataLayer.push({
+                    'data-ex-variant': 'experiment-variant-1',
+                    'data-ex-name': 'experiment-headline-variant'
+                });
+            } else if (href.indexOf('v=b') !== -1) {
```
Comment (alexgibson) [inline media/js/firefox/welcome/welcome10-experiment.js:17]:
```suggestion
            } else if (href.indexOf('v=2') !== -1) {
```

File: media/js/firefox/welcome/welcome10-experiment.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    'use strict';
+    /* update dataLayer with experiment info */
+    var href = window.location.href;
+
+    var initTrafficCop = function () {
+        if (href.indexOf('v=') !== -1) {
+            if (href.indexOf('v=a') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'experiment-variant-1',
+                    'data-ex-name': 'experiment-headline-variant'
```
Comment (alexgibson) [inline media/js/firefox/welcome/welcome10-experiment.js:15]: Should the experiment name include `welcome-10` so we know what page it relates to?

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of nits/suggestions but otherwise looks good r+wc

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of nits/suggestions but otherwise looks good r+wc 🐼

File: bedrock/firefox/templates/firefox/welcome/page10.html (hunk 3 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/welcome/page10.html:14]: I think it's only targeting North America so it's the view that needs fixing. I'll verify to be sure.

**Review by alexgibson — APPROVED** [review]
r+ 🍡

**Pull request merged**